telemetry/specs/INT.mdk

```diff
@@ -2021,6 +2042,23 @@ specific.
 * Queue drop count
   - Total # of packets dropped from the queue
 
+
+The buffer occupancy INT packet observes in the device while being forwarded. Use case is when buffer is
```

Comment:
Let's start the sentence with sth like "The metadata below are introduced to capture the buffer occupancy INT packet ... ".

Reply:
Done.